includes/modules/payment/securesubmit.php

```diff
@@ -164,50 +174,42 @@ public function process_button() {
 
     public function before_process() {
         global $_POST, $order, $sendto, $currency, $charge, $db, $messageStack; 
-        require_once(DIR_FS_CATALOG . DIR_WS_MODULES . 'payment/securesubmit/Hps.php');
+        require 'vendor/autoload.php';
```

Comment:
Once the SDK is pulled into the plugin, we need to set up an autoloader and require it here in order to load the SDK at runtime

Reply:
updated